notifications_utils/recipients.py

```diff
@@ -472,12 +474,19 @@ def validate_address(address_line, column):
     return address_line
 
 
+def get_validator(template_type: str, international_sms: bool) -> Callable:
+    if template_type == "email":
```

Comment:
same thing this should probably be a dict and you look up by key. Separate function is great but not sure if else statements are the best approach here

Reply:
Okay sounds good - looks like I can just add a type definition to the dict and that solves the type error I was trying to fix here. Will change.